daemon/mgr/spec_cgroup_cpu.go

```diff
@@ -28,3 +28,25 @@ func setupCgroupCPUSet(ctx context.Context, meta *ContainerMeta, spec *SpecWrapp
 	cpu.Mems = meta.HostConfig.CpusetMems
 	return nil
 }
+
+func setupCgroupCPUPeriod(ctx context.Context, meta *ContainerMeta, spec *SpecWrapper) error {
```

Comment:
as I said in #1031, this kind of spec implement not very good,  like this fucntion, `if s.Linux.Resources.CPU == nil`  has been judged twice here, or maybe more times in other function. 
These commit has no related with this pr.

Reply:
Thanks a lot for your comment. I totally agree with you. Spec related construction needs refactoring.